solace-message-router/nestedtemplates/workspace-disabled-shared-resources.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
remove this file after updating for the condition property

Reply:
Removed